specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2021-01-01-preview/examples/migrateFromSmartDetectionStartMigration.json

```diff
@@ -0,0 +1,24 @@
+{
+  "parameters": {
+    "subscriptionId": "70afbb35-0463-4758-badf-d5aab5bce520",
+    "api-version": "2021-01-01-preview",
+    "parameters": {
+      "scope": [
+        "/subscriptions/70afbb35-0463-4758-badf-d5aab5bce520/resourceGroups/MyRg/providers/microsoft.insights/components/myai"
+      ],
+      "actionGroupCreationPolicy": "Auto"
+    }
+  },
+  "responses": {
+    "202": {
+      "headers": {},
+      "body": {
+        "id": "ebf4f1bf-2f56-44b2-99b0-870867ae549b",
```

Comment:
>"id" [](start = 8, length = 4)

Your example should return an "id", "name" and "type" for any Azure resource

Reply:
ok. so I must return an Azure Resource?
the Id of this API is to allow users to trigger a migration process (takes couple of minutes) and then get there migration status. it's a proxy resource. and once they migration is completed we don't intend to treat this as resources, that you can delete etc...
I can return type, id and put the id that is currently returned in name. although migrationId would be clearer than name.

Reply:
at minimum "id" and "type" should be included, "id" should relate to your Azure resource Id and not your custom migrationId, you can expose migrationId inside your "properties"